packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/RNTesterReactHostDelegate.kt

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package com.facebook.react.uiapp
+
+import android.content.Context
+import com.facebook.fbreact.specs.SampleTurboModule
+import com.facebook.react.JSEngineResolutionAlgorithm
+import com.facebook.react.ReactPackage
+import com.facebook.react.ReactPackageTurboModuleManagerDelegate
+import com.facebook.react.TurboReactPackage
+import com.facebook.react.bridge.JSBundleLoader
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridgeless.BindingsInstaller
+import com.facebook.react.bridgeless.JSCInstance
+import com.facebook.react.bridgeless.JSEngineInstance
+import com.facebook.react.bridgeless.ReactHost
+import com.facebook.react.bridgeless.ReactHostDelegate
+import com.facebook.react.bridgeless.hermes.HermesInstance
+import com.facebook.react.common.annotations.UnstableReactNativeAPI
+import com.facebook.react.config.ReactFeatureFlags
+import com.facebook.react.defaults.DefaultTurboModuleManagerDelegate
+import com.facebook.react.fabric.ReactNativeConfig
+import com.facebook.react.module.model.ReactModuleInfo
+import com.facebook.react.module.model.ReactModuleInfoProvider
+import com.facebook.react.shell.MainReactPackage
+import com.facebook.react.turbomodule.core.TurboModuleManager
+import com.facebook.react.uiapp.component.MyLegacyViewManager
+import com.facebook.react.uiapp.component.MyNativeViewManager
+import com.facebook.react.uimanager.ViewManager
+
+@UnstableReactNativeAPI
+internal class RNTesterReactHostDelegate(private val context: Context) : ReactHostDelegate {
+  var reactHost: ReactHost? = null
+  private var mReactPackages: List<ReactPackage> = emptyList()
+
+  override val jSMainModulePath: String = "js/RNTesterApp.android"
+
+  override val jSBundleLoader: JSBundleLoader
+    get() =
```

Comment:
I think all those `get() =` could be removed as well. Could you verify? (applies to all of them)